api/VTTCue.json

```diff
@@ -0,0 +1,630 @@
+{
+  "api": {
+    "VTTCue": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/VTTCue",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "26",
+            "notes": "See <a href='https://bugzil.la/868509'>Bug 868509</a>."
```

Comment:
Ideally this note would have some actual information beyond just the bug link. Something appropriate based on the bug contents, like "Fully implemented except for the parts involving alien technology (see bug xxxxx)."